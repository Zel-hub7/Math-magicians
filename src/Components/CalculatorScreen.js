import React from 'react';
import './calculator.css';

function CalculatorScreen({value}) {
  return (
    <input className="calculator-screen" type="text" value={value} disabled />
  );
}

export default CalculatorScreen;
