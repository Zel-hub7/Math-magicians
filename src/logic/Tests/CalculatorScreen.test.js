import React from 'react';
import { render } from '@testing-library/react';
import CalculatorScreen from '../../Components/CalculatorScreen';

test('CalculatorScreen snapshot', () => {
  const { asFragment } = render(<CalculatorScreen value="123" />);
  expect(asFragment()).toMatchSnapshot();
});
