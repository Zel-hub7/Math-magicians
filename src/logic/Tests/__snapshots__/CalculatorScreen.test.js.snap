// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorScreen snapshot 1`] = `
<DocumentFragment>
  <input
    class="calculator-screen"
    disabled=""
    type="text"
    value="123"
  />
</DocumentFragment>
`;
